docs/docs/using-gatsby-image.md

```diff
@@ -0,0 +1,20 @@
+Using gatsby-image to prevent image bloat
+====
+
+I recently introduced a new Gatsby/React component called [gatsby-image](https://www.gatsbyjs.org/packages/gatsby-image/).
```

Comment:
To avoid saying "I" here, maybe just turn this sentence into an introduction to the article. I just realized you could copy the introduction to the plugin READme file here: You know what I just realized! I ought to also have included the plugin READme file as a suggestion for what to include in this doc. I think the intro and then the "Problem" and "Solution" sections would be great to include in this doc: https://www.gatsbyjs.org/packages/gatsby-image/

Also somehow keeping the link to the plugin READme file would be great :)